engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -1756,8 +1759,8 @@ private Option<IdentifiedLocation> getIdentifiedLocation(Token ast, boolean gene
     return Option.apply(switch (ast) {
       case null -> null;
       default -> {
-        int begin = Math.toIntExact(ast.getStartCode());
-        int end = Math.toIntExact(ast.getEndCode());
+        int begin = (int)(ast.getStartCode());
```

Comment:
Are we sure this is safe? I have a vague memory of rewriting some of the explicit casts to `int` to `Math.toIntExact` in this file. History of these lines do not reveal anything interesting though.